tools/ci-build/runtime-versioner/src/main.rs

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::{
+    repo::Repo,
+    tag::{previous_release_tag, release_tags},
+};
+use anyhow::Result;
+use camino::Utf8PathBuf;
+use clap::Parser;
+use tracing_subscriber::{filter::LevelFilter, EnvFilter};
+
+mod audit;
+mod command;
+mod index;
+mod repo;
+mod tag;
+mod util;
+
+#[derive(clap::Args, Clone)]
+pub struct Audit {
+    /// Don't `git fetch` before auditing.
+    #[arg(long)]
+    no_fetch: bool,
+    /// Explicitly state the previous release's tag. Discovers it if not provided.
+    #[arg(long)]
+    previous_release_tag: Option<String>,
+    /// Path to smithy-rs. Defaults to current working directory.
+    #[arg(long)]
+    smithy_rs_path: Option<Utf8PathBuf>,
+    /// (For testing) Path to a fake crates.io index.
```

Comment:
are there instructions somewhere for how to create this?

Reply:
It's a simple toml file of package name to version number. There are examples in the tests.